pkg/eventingtls/servermanager_test.go

```diff
@@ -0,0 +1,142 @@
+//go:build !race
+// +build !race
```

Comment:
The tests were giving race warnings due to: 
- `recv.UpdateHandler(handler)` in message_receiver
- `s.httpReceiver.UpdateHandler(s.handler)` in servermanager
- `drain.Drain()` in message_receiver

The server is calling drain after an update to that field.